<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Sample Suite" duration-ms="281" started-at="2020-04-20T20:01:03Z" finished-at="2020-04-20T20:01:03Z">
    <groups>
      <group name="smoke">
        <method signature="LoginTest.loginTest()[pri:0, instance:Tests.LoginTest@1753acfe]" name="loginTest" class="Tests.LoginTest"/>
        <method signature="LoginTest.sample()[pri:0, instance:Tests.LoginTest@1753acfe]" name="sample" class="Tests.LoginTest"/>
      </group> <!-- smoke -->
    </groups>
    <test name="Sample Test" duration-ms="281" started-at="2020-04-20T20:01:03Z" finished-at="2020-04-20T20:01:03Z">
      <class name="Tests.LoginTest">
        <test-method status="FAIL" signature="loginTest()[pri:0, instance:Tests.LoginTest@1753acfe]" name="loginTest" duration-ms="0" started-at="2020-04-21T01:31:03Z" finished-at="2020-04-21T01:31:03Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Tests.LoginTest.loginTest(LoginTest.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="sample()[pri:0, instance:Tests.LoginTest@1753acfe]" name="sample" duration-ms="0" started-at="2020-04-21T01:31:03Z" finished-at="2020-04-21T01:31:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sample -->
      </class> <!-- Tests.LoginTest -->
    </test> <!-- Sample Test -->
  </suite> <!-- Sample Suite -->
</testng-results>
